### PR TITLE
Add Codecov

### PR DESCRIPTION
**Pull request by yoavrotems**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - sudo apt-get install -y rpm
+
+script:
+  - GO111MODULE=on go test ./...
+  - go test -race -coverprofile=coverage.txt -covermode=atomic ./...
+  
+
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
+deploy:
```
Comment (danielpacak) [inline .travis.yml]: I think you can drop the deploy stanza when it comes to code cov. To configure GitHub releases it's another story and most likely a dedicated PR: you'll need goreleaser.yaml, encrypted GitHub access token, etc.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+notifications:
+  email: false
+
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install -y rpm
+
+script:
+  - GO111MODULE=on go test ./...
```
Comment (danielpacak) [inline .travis.yml]: Use `make tests` here. Travis CI has make preinstalled.

File: makefile [hunk 3]
```diff
@@ -0,0 +1,3 @@
+tests:
+	go test -race -timeout 30s -cover ./cmd ./check
```
Comment (danielpacak) [inline makefile]: This line fails. You don't have cmd and check packages. And the second go test command caters for all packages.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,10 @@
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+[![Coverage Status][cov-img]][cov]
```
Comment (danielpacak) [inline README.md]: Let's add Travis CI badge as well

File: .travis.yml (hunk 1 above)
Comment (yoavrotems) [reply .travis.yml]: OK

**Pull request merged**